ui/.neutrinorc.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
         'react/no-access-state-in-setstate': 'off',
         'babel/no-unused-expressions': 'off',
         'linebreak-style': 'off',
+        'no-underscore-dangle': 'off',
```

Comment:
Is this to allow the use of `__typename`?  Is it normal that a dataIdFromObject function has to use that property?  If so, perhaps it should just be disabled for that function?